src/docs/implementations/jmx.adoc

```diff
@@ -12,6 +12,34 @@ Micrometer also sometimes scrapes data from JMX beans for use in reporting metri
 
 include::hierarchical-name-mapping.adoc[]
 
+== Configuring
+
+The example below uses a `JmxMeterRegistry` to set values into JMX beans.
+
+[source,java]
+----
+
+meterRegistry = new JmxMeterRegistry(JmxConfig.DEFAULT, Clock.SYSTEM);
+counter = Counter.builder("app.counter.example.tick")
+        .tags("level", "normal")
+        .description("Number of info level events that made it to the logs")
+        .register(meterRegistry);
+
+counter.increment();
+
+----
+
+These values can be exposed to Prometheus using the https://github.com/prometheus/jmx_exporter[JMX Exporter].
```

Comment:
Why would I want to use the `jmx_exporter` as opposed to just exposing these metrics to Prometheus directly via `micrometer-registry-prometheus`. JMX will lose the fidelity of tags.

Reply:
Good question. I have a very customized use case. 

Remove it?